src/cli.ts

```diff
@@ -125,6 +125,8 @@ sade('lighthouse-parade <url>', true)
 
     const includePathGlob: string[] = toArray(opts['include-path-glob']);
 
+    const formFactor = opts['form-factor'];
```

Comment:
Could you also please add a check here similar to the other flags that checks that the input is a string and that it is either 'desktop' or 'mobile', and fail with a helpful error otherwise?